lib/ibm_cloud_sdk_core/base_service.rb

```diff
@@ -102,6 +103,8 @@ def request(args)
       return DetailedResponse.new(response: response) if (200..299).cover?(response.code)
 
       raise ApiException.new(response: response)
+    rescue OpenSSL::SSL::SSLError
+      raise StandardError.new("If you're trying to call a service on ICP or Cloud Pak for Data, you may not have a valid SSL certificate. If you need to access the service without setting that up, try using the disableSslVerification option in your authentication configuration and/or setting DisableSslVerification(true); on your service.")
```

Comment:
Are `DisableSslVerification(true)` and `disableSslVerification` correct for ruby?

Reply:
ah. good catch.